apps/docs/src/app/platform/component-docs/platform-forms/radio-group/platform-radio-group-examples/platform-radio-group-content-example.component.html

```diff
@@ -1,126 +1,96 @@
-<p>Reactive form Examples</p>
-<form [formGroup]="customForm" class="flex-form">
-    <fieldset fd-fieldset>
-        <p>Example 1: Compact Radio Button Group with 'default' state and with default selection</p>
-        <fdp-radio-group
-            [id]="'radio1'"
-            [name]="'radio1'"
-            [contentDensity]="'compact'"
-            [value]="'Spring'"
-            [isInline]="false"
-            formControlName="example1"
-        >
-            <fdp-radio-button *ngFor="let season of seasons" [value]="season">
-                {{ season }}
-            </fdp-radio-button>
+<h3>Use With Reactive Form</h3>
+<p>Compact Radio Button Group with 'default' State and with Default Selection</p>
+<fdp-form-group [multiLayout]="true" [formGroup]="form1">
+    <fdp-form-field #ffl1 [id]="'radio1'" zone="zLeft" rank="1">
+        <fdp-radio-group [name]="'radio1'" [contentDensity]="'compact'" [value]="'spring'" [isInline]="false"
```

Comment:
you should not pass and set name when in inside formField, you set the name on the FF level.

Reply:
Raise ISSUE againts your formGroup that we need to update this againts latest review comments we had for other component. 

Some Properties are globally configurable (Injectable) and some properties not. contentDensity is something to set globally.

Reply:
raised Issue to move name property to fdp-form-field.